File: 2st/Yeonwoo/정사각형으로 만들기_.py
def solution(arr):
    answer = []
    if len(arr)==len(arr[0]):
        answer = arr
    elif len(arr)>len(arr[0]):
        for i in range(len(arr)):     
            answer.append(arr[i]+[0]*(len(arr)-len(arr[0])))
    elif len(arr)<len(arr[0]):
        for i in range(len(arr)):
            answer.append(arr[i])
        for j in range(len(arr[0])-len(arr)):
            answer.append([0]*len(arr[0]))
    return answer

File: 4st/Eunseong/정사각형으로 만들기.java
class Solution {
    public int[][] solution(int[][] arr) {
        int maxLen = Math.max(arr.length, arr[0].length);
        int answer[][] = new int[maxLen][maxLen];
        for (int i = 0; i < arr.length; i++) {
            for (int j = 0; j < arr[i].length; j++) {
                answer[i][j] = arr[i][j];
            }
        }
        return answer;
    }
}